assets/src/edit-story/components/panels/design/animation/directionRadioInput.js

```diff
@@ -77,7 +77,7 @@ const Icon = styled.div`
     selected &&
     !disabled &&
     css`
-      background-color: #51389d;
+      background-color: #1a73e8;
```

Comment:
I think we got this `#1a73e8` in our edit-story theme provider under `theme.colors.accent.primary`

Just for context, rn we have two separate theme providers in the dashboard and edit story that are primarily under the old design language. Brittany is currently working on a new design system under the new design language that unifies these

Reply:
Oh nice, I'll just use that one then 👍